### ISSUE TITLE
10-10d v2 Staging Review finding: Buttons' accessible name does not include the text value

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#118213]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** IVC Forms
**Product:** 10-10D CHAMPVA Benefits and 10-7959c OHI
**Feature:** Merged form solution

## Findings details
**VA.gov Experience Standard - issue:** [User encounters design components or patterns that are inconsistent or confusing.](https://design.va.gov/about/experience-standards/#user-encounters-consistent-clear-design-components)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

On Review & Submit, when editing sections, the "Update page" buttons do not include the full text value of the button "update page". Instead they announce as "Update your name, button" for example. 

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="630" height="232" alt="Update page" src="https://github.com/user-attachments/assets/86e95d6e-85a6-41b7-bc56-72f89ac61256" />

</details>

## Recommended action

When additional context is provided to a button using `aria-label` or another means, the full button text should be included to ensure the visual text is aligned to the accessible name of the button. Providing the full text in the accessible name helps speech input users effectively identify the button. 

In this case, the buttons should read as "Update page [section name]". Example: "Update page - your name", so that the visual text "Update page" is represented in the accessible name. 

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** [WCAG 2.5.3 Label in Name - Level A](https://www.w3.org/TR/WCAG22/#label-in-name)
- **Modality:** Voice
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by andrea-merrill** [comment]
closing as a duplicate of: [10-10d v2 Staging Review finding: Buttons inconsistently labelled for assistive technologies](https://github.com/department-of-veterans-affairs/va.gov-team/issues/118212)

this is an issue with the review page, which is owned by the design system
